.github/workflows/executable.yml

```diff
@@ -20,23 +20,20 @@ jobs:
           APPLE_DEV_CERT: ${{secrets.APPLE_DEV_CERT}}
           APPLE_ID_USERNAME: ${{secrets.APPLE_ID_USERNAME}}
           APPLE_ID_KEY: ${{secrets.APPLE_ID_KEY}}
-      - name: Create tar files
+      - name: Create zip files
         run: |
-          tar -cvf percy-linux.tar percy
+          zip percy-linux.zip percy
```

Comment:
why not `tar.gz` file since zip looses the executable permission?

```bash
tar -czvf percy-linux.tar.gz percy`
```

Reply:
it does not in this case. it was loosing when we were uploading as artifact.